src/intent.html

```diff
@@ -221,20 +221,14 @@ <h3 id="mixing_intent_properties">Intent Properties</h3>
    but as with any properties, is by default ignored but may have a
    system-specific effect.</p>
 
-    <p>As with [=Concepts=], The Working group maintains two lists
-      of [=property=] values.</p>
+    <p>Similar to the lists of [=Concepts=], The Working Group maintains a list of [=property=] values that should be implemented.</p>
     <ul>
-      <li><dfn id="intent-core-properties">Core properties</dfn>: This is a
-      <a href="https://w3c.github.io/mathml-docs/intent-core-properties">list core of properties</a>
+      <li><dfn id="intent-core-properties">Properties list</dfn>: This is the full
```

Comment:
what does `full` mean

Reply:
it was the wording used before, the full list is maintained in the yaml and an "edited highlights" appears here. Actually if the full list doesn't get much bigger  it may make sense to have it all here, but for now I'll merge this with the following paragraph and re-word. The version here has too much text left over from describing two lists